scripts/travis/build_test.sh

```diff
@@ -16,7 +16,7 @@ SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
 if [ "${USER}" = "travis" ]; then
     # we're running on a travis machine
     "${SCRIPTPATH}/build.sh" --make_debug
-    "${SCRIPTPATH}/travis_wait.sh" 90 "${SCRIPTPATH}/test.sh"
+    "${SCRIPTPATH}/travis_wait.sh" 120 "${SCRIPTPATH}/test.sh"
```

Comment:
do we have any need for that ?

Reply:
No, I was copying over the value from the other script. I'll switch it back.